docs_src/design/adr/device-service/0012-DeviceService-Filters.md

```diff
@@ -0,0 +1,83 @@
+# Device Service Filters
+
+## Status
+
+### Proposed
+
+- design for Hanoi
+- implementation for Ireland
+
+## Context
+
+In EdgeX today, sensor/device data collected can be "filtered" by [application services](../../../microservices/application/ApplicationServices.md) before being exported or sent to some [north side](../../../general/Definitions.md#south-and-north-side) application or system. Built-in application service functions (available through the app services SDK) allow EdgeX event/reading objects to be filtered by device name or by value descriptor type.  That is, event/readings can be filtered by:
+
+- which device sent the event/reading (as determined by the device name in the event/readings)
+- the category (such as temperature or humidity) of data as determined by the value descriptor.

```

Comment:
Change `determined by the value descriptor` to `determined by the Reading's value descriptor name. aka Reading Name`,